docs/checks/internal/checks.yaml

```diff
@@ -746,6 +746,7 @@ checks:
       example, by using build scripts controlled by the author of the PR or reading
       token in memory. This check does not detect whether untrusted code checkouts are
       used safely, for example, only on pull request that have been assigned a label.
+      See [Keeping your GitHub Actions and workflows secure Part 1](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/).
```

Comment:
we already link to this a few lines below in the remediation section. 
```
See this [post](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/)
        for information on avoiding untrusted code checkouts.
```

Is this a matter of visibility? having the link in one section vs another?

Reply:
Deleted duplicate link and re-pushed.

Reply:
The PR now appears to be a single tab. 
Can the PR and issue be closed? Or is there still some unaddressed ambiguity? I'm all for fixing ambiguity in the docs if needed.

Reply:
We ended up using this pull request to complete the deferred conversation at https://github.com/ossf/scorecard/issues/2831#issuecomment-1504090074 . Let's close this.